lib/Conversion/FIRRTLToHW/LowerToHW.cpp

```diff
@@ -1063,6 +1063,11 @@ FIRRTLModuleLowering::lowerModule(FModuleOp oldModule, Block *topLevelModule,
   if (auto comment = oldModule->getAttrOfType<StringAttr>("comment"))
     newModule.setCommentAttr(comment);
 
+  // Pass along the number of random initialization bits needed for this module.
+  if (auto randomWidth =
+          oldModule->getAttrOfType<IntegerAttr>("firrtl.random_init_width"))
+    newModule->setAttr("firrtl.random_init_width", randomWidth);
```

Comment:
It's kind of annoying that we have to do this explicitly as opposed to having all attributes automatically copied over to their HW counterparts.

It's inappropriate to handle this in this PR, though. (Keep your attribute copying approach.)

Reply:
👍  there are some new-ish hooks for handling all dialect attributes at once, which LowerSeqToSV uses. We could do that here as well.